CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# dbt_hubspot_source v0.18.1
```

Comment:
Very small update request. Can you also update the version [here](https://github.com/kcraig-ats/dbt_hubspot_source/blob/4915369915a86035e65bf3b903dda7bae1ab5cec/dbt_project.yml#L2) and [here](https://github.com/kcraig-ats/dbt_hubspot_source/blob/4915369915a86035e65bf3b903dda7bae1ab5cec/integration_tests/dbt_project.yml#L2) to reflect the new patch update.